### PR TITLE
Libhandy v0.0.6: added package

### PR DESCRIPTION
**Pull request by tallero**
Stable version with fix for MinGW, out some hour ago.

### CODE REVIEW

**Comment by mati865** [comment]
Looks like you forgot to add non git package in https://github.com/Alexpux/MINGW-packages/pull/4783/commits/78f7933236b552758895b9501dc41fc5fdf8d718.

File: mingw-w64-libhandy/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+url="https://source.puri.sm/Librem5/libhandy"
+license=("LGPL2.1")
+arch=('any')
+depends=("${MINGW_PACKAGE_PREFIX}-gtk3")
+makedepends=("${MINGW_PACKAGE_PREFIX}-meson"
+             "${MINGW_PACKAGE_PREFIX}-pkg-config"
+             "${MINGW_PACKAGE_PREFIX}-vala"
+             "${MINGW_PACKAGE_PREFIX}-gobject-introspection")
+options=('strip' 'staticlibs' '!debug')
+source=('https://source.puri.sm/Librem5/libhandy/-/archive/v0.0.6/libhandy-v0.0.6.tar.gz')
```
Comment (Alexpux) [inline mingw-w64-libhandy/PKGBUILD]: Dont hardcode version. Use ````pkgver```` variable instead

Comment (tallero) [reply mingw-w64-libhandy/PKGBUILD]: done

**Comment by tallero** [comment]
@Alexpux these are the files installed by the package:
    mingw-w64-x86_64-libhandy /mingw64/
    mingw-w64-x86_64-libhandy /mingw64/include/
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/handy.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-action-row.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-arrows.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-column.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-combo-row.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-dialer-button.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-dialer-cycle-button.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-dialer.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-enum-value-object.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-enums.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-expander-row.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-fold.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-header-group.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-leaflet.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-list-box.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-main.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-search-bar.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-string-utf8.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-title-bar.h
    mingw-w64-x86_64-libhandy /mingw64/include/libhandy-0.0/hdy-version.h
    mingw-w64-x86_64-libhandy /mingw64/lib/
    mingw-w64-x86_64-libhandy /mingw64/lib/girepository-1.0/
    mingw-w64-x86_64-libhandy /mingw64/lib/girepository-1.0/Handy-0.0.typelib
    mingw-w64-x86_64-libhandy /mingw64/lib/libhandy-0.0-0.dll
    mingw-w64-x86_64-libhandy /mingw64/lib/libhandy-0.0.dll.a
    mingw-w64-x86_64-libhandy /mingw64/lib/pkgconfig/
    mingw-w64-x86_64-libhandy /mingw64/lib/pkgconfig/libhandy-0.0.pc
    mingw-w64-x86_64-libhandy /mingw64/share/
    mingw-w64-x86_64-libhandy /mingw64/share/gir-1.0/
    mingw-w64-x86_64-libhandy /mingw64/share/gir-1.0/Handy-0.0.gir
    mingw-w64-x86_64-libhandy /mingw64/share/vala/
    mingw-w64-x86_64-libhandy /mingw64/share/vala/vapi/
    mingw-w64-x86_64-libhandy /mingw64/share/vala/vapi/libhandy-0.0.deps
    mingw-w64-x86_64-libhandy /mingw64/share/vala/vapi/libhandy-0.0.vapi

What is the dll in the wrong path? The library is working correctly for me.

**Comment by Alexpux** [comment]
DLL must be in bin not lib

**Pull request merged**